x/funders/keeper/logic_funders.go

```diff
@@ -68,24 +95,25 @@ func (k Keeper) ChargeFundersOfPool(ctx sdk.Context, poolId uint64, recipient st
 		})
 	}
 
-	// Move funds to pool module account
-	if !payouts.IsZero() {
-		if err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, recipient, payouts); err != nil {
-			return sdk.NewCoins(), err
-		}
+	if payouts.IsZero() {
+		return
 	}
 
+	// Move funds to recipient module
+	err = k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, recipient, payouts)
```

Comment:
I really dont like this style. In case an error happen, one would return an payout amount != default and err. Usually it should be {default-value}, err

Reply:
Also if one would add another check later, one would also need to adjust this line anyways and add an if-null check. Because otherwise this error would get overwritten.